aten/src/ATen/native/native_functions.yaml

```diff
@@ -185,6 +185,15 @@
   dispatch:
     CompositeExplicitAutograd: sym_constrain_range
 
+
+- func: _same_storage(Tensor(a!) self, Tensor b) -> bool
```

Comment:
These are really bad....
Why are they all inplace?
What do they mean for non cpu/cuda Tensor?

I don't think we want such functions here becoming part of our public API (the whole IR is public API).

Reply:
Well, they are underscored, hence private/subject to change. I agree they are not great. How would you like to see them changed? I'd rather we not teach dynamo about storage or data_ptr if we don't have to. 

Alternatively, if we dont want these as native, I can add utils in FSDP and if dynamo sees them, we can turn them into a raw call_function() in the graph. Would you prefer that?

Reply:
yeah wtf why did I do this